export.html

```diff
@@ -30,14 +30,49 @@
 <p>
     The following matrix details the levels at which the different export
     formats are enabled.
+    The * indicates usage of the Batch Image Export script (see below).
+    The download of original data can be enabled or disabled for
+    Images or Plates by <a href="https://docs.openmicroscopy.org/latest/omero/sysadmins/config.html#omero-policy-binary-access">sysadmin configuration</a>.
```

Comment:
Would add a sentence: ``The table below shows the expected behaviour with the default sysadmin configuretion.``